spec/services/exports/export_distributions_csv_service_spec.rb

```diff
@@ -1,63 +1,155 @@
+          issued_at: "01/01/2025",
+          created_at: "01/01/2025",
+          delivery_method: "shipped",
+          shipping_cost: "12.09",
+          state: "scheduled",
+          agency_rep: "",
+          comment: "comment 1",
+          line_items: [
+            {
+              item: duplicate_item,
+              quantity: 5
+            },
+            {
+              item: create(:item, name: "A Item", value_in_cents: 1000, organization: organization),
+              quantity: 7
+            },
+            {
+              item: duplicate_item,
+              quantity: 3
+            }
+          ],
+          filtered_quantity: 8,
+          filtered_total_value: Money.new(2400).to_f
+        },
+        {
+          issued_at: "02/02/2025",
+          created_at: "02/02/2025",
+          delivery_method: "shipped",
+          shipping_cost: "13.09",
+          state: "scheduled",
+          agency_rep: "",
+          comment: "comment 2",
+          line_items: [
+            {
+              item: create(:item, name: "B Item", value_in_cents: 2000, organization: organization),
+              quantity: 1
+            }
+          ],
+          filtered_quantity: 0,
+          filtered_total_value: Money.new(0).to_f
+        },
+        {
+          issued_at: "03/03/2025",
+          created_at: "03/03/2025",
+          delivery_method: "shipped",
+          shipping_cost: "14.09",
+          state: "scheduled",
+          agency_rep: "",
+          comment: "comment 3",
+          line_items: [
+            {
+              item: create(:item, name: "C Item", value_in_cents: 3000, organization: organization),
+              quantity: 2
+            }
+          ],
+          filtered_quantity: 0,
+          filtered_total_value: Money.new(0).to_f
+        },
+        {
+          issued_at: "04/04/2025",
+          created_at: "04/04/2025",
+          delivery_method: "shipped",
+          shipping_cost: "15.09",
+          state: "scheduled",
+          agency_rep: "",
+          comment: "comment 4",
+          line_items: [
+            {
+              item: create(:item, name: "E Item", value_in_cents: 4000, organization: organization),
+              quantity: 3
+            }
+          ],
+          filtered_quantity: 0,
+          filtered_total_value: Money.new(0).to_f
+        }
       ]
-    end
-
-    let(:item_names) { items_lists.flatten(1).map(&:first).map(&:name).sort.uniq }
+    }
 
     let(:distributions) do
-      start_time = Time.current
-
-      items_lists.each_with_index.map do |items, i|
+      expected_distributions.each_with_index.map do |dist, i|
         distribution = create(
           :distribution,
-          issued_at: start_time - i.days, delivery_method: "shipped", shipping_cost: "12.09",
-          organization: organization
+          partner: partner,
+          organization: organization,
+          storage_location: storage_location,
+          created_at: dist[:created_at],
```

Comment:
Is there a reason you can't specify the created\_at time here? If you use `freeze_time` or `travel_to` you should be able to know these values in advance.